src/grammar.pegjs

```diff
@@ -75,20 +75,20 @@ AbsolutePath
 }
 
   Root
-    = PosixRoot
-    / WindowsRoot
+    = WindowsRoot
+    / PosixRoot
```

Comment:
I changed the order so it tries to parse Windows paths first, so that `/C:/` is recognized as a windows drive letter.

Reply:
Hm, technically this is a breaking change, but I believe most of users do not have a directory matching `/[A-Za-z]:/` under root.

Reply:
Yup